src/opm/parser/eclipse/EclipseState/Aquancon.cpp

```diff
@@ -113,10 +119,51 @@ namespace Opm {
     }
 
 
+    bool Aquancon::cellInsideReservoir(const Opm::EclipseGrid& grid, const int i, const int j, const int k)
+    {
+        if ( i < 0 || j < 0 || k < 0
+            || size_t(i) > grid.getNX() - 1
+            || size_t(j) > grid.getNY() - 1
+            || size_t(k) > grid.getNZ() - 1 )
+        {
+            return false;
+        }
+
+        return grid.cellActive(i, j, k );
+    }
+
+    bool Aquancon::neighborCellInsideReservoir(const Opm::EclipseGrid& grid,
+           const int i, const int j, const int k, const Opm::FaceDir::DirEnum faceDir)
+    {
+        switch(faceDir) {
+        case FaceDir::XMinus:
+            return cellInsideReservoir(grid, i - 1, j, k);
+            break;
+        case FaceDir::XPlus:
+            return cellInsideReservoir(grid, i + 1, j, k);
+            break;
+        case FaceDir::YMinus:
+            return cellInsideReservoir(grid, i, j - 1, k);
+            break;
```

Comment:
probably I should remove the `break`s. I will remove them along with other comments.